lib/hllcounter.cc

```diff
@@ -32,20 +32,14 @@ std::map<int, std::vector<double> > rawEstimateData;
 std::map<int, std::vector<double> > biasData;
 
 
-double get_alpha(const int p)
+double calc_alpha(const int p)
 {
-    if ((p < 4) or (p > 16)) {
-        double valid_lower_bound = 1.04 / std::sqrt(std::pow(static_cast<float>(2),
-                                   17));
-        double valid_upper_bound = 1.04 / std::sqrt(std::pow(static_cast<float>(2), 3));
-
-        std::stringstream message;
-        if (p < 4) {
-            message << "Max error should be smaller than " << valid_upper_bound;
-        } else {
-            message << "Min error should be greater than " << valid_lower_bound;
-        }
-        throw khmer_exception(message.str().c_str());
+    if (p < 4) {
+        // ceil(log2((1.04 / x) ^ 2)) = 4, solve for x
+        throw khmer_exception("Error rate must be smaller than 0.367696");
```

Comment:
This is how it shows up in Python:

```python
In [1]: import khmer
In [2]: khmer.HLLCounter(0.5, 32)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-2-d2705791ce3b> in <module>()
----> 1 khmer.HLLCounter(0.5, 32)

ValueError: Error rate must be smaller than 0.367696

In [3]: khmer.HLLCounter(0.000000001, 32)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-3-67d189173ec1> in <module>()
----> 1 khmer.HLLCounter(0.000000001, 32)

ValueError: Error rate must be greater than 0.0040624
```
Two options:
- Change the error message to be something like "Please set error rate to a value smaller than 0.367696"
- Force it to be at the limit (either upper or lower) and only emit a warning.